Lib/fontParts/base/bPoint.py

```diff
@@ -391,21 +399,21 @@ def round(self):
                        normalizers.normalizeVisualRounding(y))
 
 
-def relativeBCPIn(anchor, BCPIn):
+def relativeBCPIn(anchor: CoordinateType, BCPIn: CoordinateType) -> CoordinateType:
```

Comment:
I'm not sure how these should be annotated. I'll look into it and report back if no-one else beats me to it.

Reply:
It loks to me like these are on point.